layouts/shortcodes/clients.html

```diff
@@ -12,15 +12,32 @@ <h2 class="font-bold text-2xl sm:text-4xl text-center mb-10">
       </span>
     </h2>
     {{ end }}
-    <ul class="flex flex-col md:flex-row md:flex-wrap items-center lg:justify-between gap-12 md:gap-8">
-      {{ range .clients_item}}
-      {{ if .show }}
-      <li>
-        <img loading="lazy" class="max-w-[180px]" src="{{ .image | absURL }}" alt="{{ .name }}">
-      </li>
-      {{ end }}
-      {{ end }}
-    </ul>
+    <div class="embla overflow-hidden" data-loop="true" data-scroll="1">
```

Comment:
When the user scrolls through the logos, only 1 moves at a time. We can change this to another number. Also, the user will be able to loop through all of the logos. This means that when the user gets to the last logo they will see the first one. This matches how we handle the testimonials.